README.md

```diff
@@ -1268,9 +1268,9 @@ kod
 **[⬆ do góry](#table-of-contents)**
 
 
-## Eventy
+## Zdarzenia
```

Comment:
Wszystko superaśnie! Ale "Zdarzenia" to imho przesada :P Event to event, zdarzenie kojarzy się z interakcją z aplikacją, "czymś co ktoś zrobił". A przy modułowych aplikacjach eventem może być równie dobrze prosty callback pomiędzy modułami.